cmd/scaffold.go

```diff
@@ -858,7 +858,7 @@ func (fnb *FlowNodeBuilder) initFvmOptions() {
 		fvm.WithBlocks(blockFinder),
 		fvm.WithAccountStorageLimit(true),
 	}
-	if fnb.RootChainID == flow.Testnet || fnb.RootChainID == flow.Stagingnet || fnb.RootChainID == flow.Mainnet {
+	if fnb.RootChainID == flow.Testnet || fnb.RootChainID == flow.Stagingnet || fnb.RootChainID == flow.Localnet || fnb.RootChainID == flow.Benchnet || fnb.RootChainID == flow.Mainnet {
```

Comment:
we can just remove this all together / enable fees everywhere?

Reply:
@ramtinms @janezpodhostnik @j1010001 how do you feel about this suggestion?  Or should I just negate the condition to something like `!= Emulator`?

Reply:
There is actually a task for enabling this by default in the emulator: https://github.com/onflow/flow-cli/pull/581

Maybe it is time to just enable them everywhere by default...